app/views/help/verbal_ntp_requests.md

```diff
@@ -0,0 +1,5 @@
+<% title "How do I submit an verbal NTP request in case of emergency?" %>
```

Comment:
My assumption is this the training to get a purchase card would include a verbal notice to proceed and they'd know. But I can't imagine they _wouldn't_ know if spelled out, so I changed that.

I know how to handle the NCR/non-NCR distinction was something @phirefly was figuring out. If the resolution makes this item impractical, I can remove for now.